components/browser/menu/src/main/java/mozilla/components/browser/menu/view/ExpandableLayout.kt

```diff
@@ -360,6 +371,25 @@ internal class ExpandableLayout private constructor(context: Context) : FrameLay
         return result
     }
 
+    /**
+     * In a dynamic menu - one in which items or their positions may change the adapter position and
+     * the RecyclerView position for the same item may differ.
+     * This method helps reconcile that.
+     *
+     * @return the RecyclerView position for the item at the [adapterIndex] in the adapter or
+     * [RecyclerView.NO_POSITION] if there is no child for the indicated adapter position.
+     */
+    @VisibleForTesting
+    internal fun getChildPositionForAdapterIndex(listView: RecyclerView, adapterIndex: Int): Int {
```

Comment:
Thought about having this functionality in a custom getter for `lastVisibleItemIndexWhenCollapsed` and `stickyItemIndex` but then would loose any way of knowing the real passed in value.